pymessenger/bot.py

```diff
@@ -7,45 +7,26 @@ def __init__(self, access_token):
         self.base_url = "https://graph.facebook.com/v2.6/me/messages?access_token={0}".format(access_token)
 
     def send_text_message(self, recipient_id, message):
-        recipient = {
-                "id": recipient_id
-                }
-        message = {
-                "text": message
-                }
-        payload ={}
-        payload['recipient'] = "{0}".format(recipient)
-        payload['message'] = "{0}".format(message)
-        return requests.post(self.base_url, data=(payload)).json()
+        payload = {'recipient': {'id': recipient_id},
+                   'message': {'text': message}
+                  }
+        return requests.post(self.base_url, json=payload).json()
 
     def send_message(self, recipient_id, message):
-        recipient = {
-                "id": recipient_id
-                }
-        payload ={}
-        payload['recipient'] = "{0}".format(recipient)
-        payload['message'] = "{0}".format(message)
-        return requests.post(self.base_url, data=(payload)).json()
+        payload ={'id': recipient_id,
```

Comment:
Thanks for the pull request!

Will be merging it in, but I think we should be nesting the recipient id, under the 'recipient' key like your other changes.

```
payload ={'recipient': {'id': recipient_id}}
```

This particular function is returning the following error when running:
```

> {'error': {'code': 100, 'type': 'OAuthException', 'fbtrace_id': 'GsO0h9B2i/A', 'message': '(#100) Specifying multiple ids with a post method is not supported'}}